zebra/zebra_vrf.h

```diff
@@ -115,6 +117,11 @@ struct zebra_vrf {
 	/* l3-vni info */
 	vni_t l3vni;
 
+	/*
+	 * Flooding mechanism for BUM packets for VxLAN-EVPN.
+	 */
+	enum vxlan_flood_control vxlan_flood_ctrl;
+
```

Comment:
I would like to understand why this provisioning in zebra_vrf abstraction.
- one the one hand, using default vrf, vni discovery happens only at default vrf.
so if the command is triggered on non default vrf, zebra will refuse it
- on the other hand, this may be usefule for vrf netns backend. 

May I know the reason why this vni context has been nested into vrf?

Reply:
To clarify a off line discussion @pguibert6WIND and I have had:  This was just the way it was done originally and we both agree that this needs to be reworked some.  My proposal was that this is going to be done as part of the zebra_router work I am slowly doing.